dashboard.py

```diff
@@ -98,6 +104,9 @@ def make_timeseries(originalname, df, spec) -> altair.Chart | None:
                 altair.Tooltip("dask_version:N", title="Dask"),
                 altair.Tooltip(f"{spec.field}:Q", title=spec.label),
                 altair.Tooltip("ci_run_url:N", title="CI Run URL"),
+                altair.Tooltip(
+                    "performance_report_url:N", title="Performance Report URL"
+                ),
```

Comment:
I'm not sure if the tooltip will help all that much because we cannot click on the tooltips. Another delivery medium might be better suited.
In a perfect world, clicking would open a dedicated page for that test with additional info, etc.

Reply:
https://altair-viz.github.io/gallery/scatter_href.html

Reply:
href already point to the CI job